app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -95,13 +95,15 @@ private void initView() {
     }
 
     private void renderContributors(List<ContributorViewModel> contributors) {
-        adapter.addAllWithNotify(contributors);
+        adapter.clear();
+        adapter.addAll(contributors);
+        adapter.notifyDataSetChanged();
```

Comment:
I have found another idea to use ObservableList in databinding package.
https://developer.android.com/reference/android/databinding/ObservableList.html

I will try either DataUtil or ObservableList on another PR so please ignore this time.